### PR TITLE
[add] status msg rendering via JS

### PR DESCRIPTION
**Pull request by simahawk**
Feature explained in updated README ;)

### CODE REVIEW

File: cms_status_message/static/src/xml/status_message.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8" ?>
+
+<templates id="qweb_templates" xml:space="preserve">
```
Comment (simahawk) [inline cms_status_message/static/src/xml/status_message.xml]: Here I'm forced to duplicate the templates as I don't know an easy way to reuse original ir.ui.view templates. I cannot find any docs. Just spotted this approach tough: https://github.com/odoo/odoo/blob/9.0/addons/website/static/src/js/website.theme.js#L21

**Comment by simahawk** [comment]
tests are failing but the fix is already included into https://github.com/OCA/website-cms/pull/8 that is ready to be merged since a while.

**Review by guewen — APPROVED** [review]

**Comment by simahawk** [comment]
Waiting merge of OCA/website-cms/pull/11 to disable transifex

**Comment by simahawk** [comment]
@lasley if you are still around, pls, hit the button! :smile:

**Review by lasley — APPROVED** [review]

**Pull request merged**